progress/angular/projects/ng-progress-icons/package.json

```diff
@@ -4,11 +4,13 @@
     "description": "Angular progress icons for PX Blue applications",
     "main": "bundles/pxblue-ng-progress-icons.umd.js",
     "devDependencies": {
+        "color": "^3.1.2",
         "tslib": "^1.9.0"
     },
     "peerDependencies": {
         "@angular/common": ">=6.0.0",
-        "@angular/core": ">=6.0.0"
+        "@angular/core": ">=6.0.0",
+        "color": "^3.1.2"
```

Comment:
I would make this a dependency instead of peerDependency so the user doesn't have to be responsible for installing it. We have this as a direct dependency for our component libraries.

Reply:
Okay I'll move this over so it matches other libraries.

Reply:
I'm seeing this after making colors a dependencies instead of a peerDependency. 

```
$ ng build ng-progress-icons
Building Angular Package
Building entry point '@pxblue/ng-progress-icons'
Compiling TypeScript sources through ngc
Browserslist: caniuse-lite is outdated. Please run next command `yarn upgrade`
Bundling to FESM2015
Bundling to FESM5
Bundling to UMD
No name was provided for external module 'color' in output.globals – guessing 'color'
Minifying UMD bundle
Copying declaration files
Writing package metadata
Distributing npm packages with 'dependencies' is not recommended. Please consider adding color to 'peerDependencies' or remove it from 'dependencies'.

BUILD ERROR
Dependency color must be explicitly whitelisted.
An unhandled exception occurred: Dependency color must be explicitly whitelisted.

```

Reply:
I think we should listen to the builder's error.  Also I've already defensively coded usage of the `color` library so our icon won't crash if this dependency is missing.

```
 if (color) {
     // Use color library
}
```